recipe/meta.yaml

```
@@ -1,45 +1,58 @@
 {% set name = 'freetds' %}
-{% set version = "1.1.15" %}
+{% set version = "1.3.9" %}
+{% set sha256 = "a81c8a986a7a2f0b94599ebd7a1785d72017dea6d796a7f44b6ac681ee77c2e7" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: ftp://ftp.freetds.org/pub/freetds/stable/freetds-{{ version }}.tar.gz
-  sha256: ebeb9060bcfe54f13ab0fbf8cb72ac95cc6cefbe728a30247516d55334d66a03
+  fn: freetds-{{ version }}.tar.gz
+  url: http://www.freetds.org/files/stable/{{ name }}-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
```

Don't add the indirection
```suggestion
  sha256: a81c8a986a7a2f0b94599ebd7a1785d72017dea6d796a7f44b6ac681ee77c2e7
```